src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -703,10 +705,13 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
           required
           label={t('create_single_item_modal.category_label')}
           placeholder={t('create_single_item_modal.category_placeholder')}
-          value={categories.includes(category!) ? category : undefined}
+          value={isValidCategory ? category : undefined}
           options={categories.map(value => ({ value, text: t(`${type!}.category.${value}`) }))}
           onChange={this.handleCategoryChange}
         />
+        {type === ItemType.WEARABLE && isValidCategory && !isValidWearableMetrics(metrics as ModelMetrics, category as WearableCategory) ? (
```

Comment:
do we need this check `isValidWearableMetrics` here? The `ErrorMetrics` will also do it, right? looks like it will be validated twice

Reply:
You're right, the `ErrorMetrics` do the same validation in parts to show `textures` errors or `triangles` errors. 
The reason for the validation here is to avoid rendering the component container.